qcengine/programs/tests/test_standard_suite.py

```diff
@@ -33,6 +33,7 @@
 
 import pytest
 import qcelemental as qcel
+import sys
```

Comment:
ah yeah, I want that fn to fail at test-design time, and it fails equally well with "no sys" as "sys.exit". thanks.

Reply:
no problem!